cases/sbml

```diff
@@ -0,0 +1 @@
+.
```

Comment:
?

Reply:
symlink `cases/sbml` -> `cases/` instead of moving the cases directories to `cases/sbml` and breaking backward compatibility ;)

Reply:
ahh. Yes, I though this was supposed to be somewhere, but github shows this weirdly.